prefetch.bs

```diff
@@ -963,6 +975,74 @@ The following parameters are defined for the `prefetch` token:
     This specification conforms to this advice; the [=start a referrer-initiated navigational prefetch=] algorithm does not emit non-boolean values.
   </div>
 
+<h2 id="automated-testing">Automated testing</h2>
+
+For the purposes of user-agent automation and application testing, this document defines extensions to the [[WebDriver-BiDi]] specification.
+
+<div class="issue">CDDL snippets use the "text" type instead of "browsingContext.BrowsingContext" to allow indepedent programmatic processing of CDDL snippets. Currently, other modules cannot be referenced.</div>
+
+<h3 id="speculation-module">The speculation module</h3>
+
+The speculation module contains commands for managing the remote end behavior for prefetches, prerenders, and speculation rules.
+
+<h3 id="speculation-module-definitions">Definitions</h3>
+
+{^Remote end definition^} and {^local end definition^}
+
+<pre class="cddl" data-cddl-module="remote-cddl,local-cddl">
+speculation.PreloadingStatus = "pending" / "ready" / "success" /
+                   "failure" / "not-supported"
+</pre>
+
+The <code>speculation.PreloadingStatus</code> type represents the different types of preloading statuses. This status is shared by prefetches and prerenders.
+
+"<dfn for="speculation.PreloadingStatus" cddl-value>pending</dfn>" status is defined when a [=prefetch record=] |prefetchRecord| begins to [=start a referrer-initiated navigational prefetch=].
+
+"<dfn for="speculation.PreloadingStatus" cddl-value>ready</dfn>" status is defined when a [=prefetch record=] |prefetchRecord| is [=prefetch record/complete=].
+
+"<dfn for="speculation.PreloadingStatus" cddl-value>success</dfn>" status is defined when a [=prefetch record=] |prefetchRecord| is activated, such as from the user agent's navigation.
+
+"<dfn for="speculation.PreloadingStatus" cddl-value>failure</dfn>" status is defined when a [=prefetch record=] |prefetchRecord| has failed, such as when the |prefetchRecord| expires.
+
+"<dfn for="speculation.PreloadingStatus" cddl-value>not-supported</dfn>" status is defined when a [=prefetch record=] |prefetchRecord| is ineligble to [=start a referrer-initiated navigational prefetch=].
```

Comment:
Nothing in this spec ever sets this status. Is that still work to be done?

Reply:
Looking through the implementation, I don't see any places currently in the spec where this would match. It's mainly related to prerendering logic, like upgrading to prerender or having a duplicate preload attempt. I'm thinking we could remove this value from the spec, how does that sound?

Reply:
Yes, sounds good, let's remove it!